src/main/java/org/training/poker/Hand.java

```diff
@@ -70,6 +132,25 @@ public Ranking getRanking() {
 		return rankFrequency;
 	}
 
+	public Map<Integer, List<Rank>> frequencyToRank() {

```

Comment:
frequencyToRank() is only used by compareHands(): it is only used by its own class, therefore it could have been hidden/made private.

Reply:
it needs to be default or protected, so the unit test class can access it